### PR TITLE
feat: update plugins to Flicking v4

### PR DESCRIPTION
**Pull request by WoodNeck**
## Details
This updates the plugins' interface to match a newer specification of Flicking.

### CODE REVIEW

File: src/Fade.ts [hunk 1]
```diff
@@ -1,45 +1,69 @@
...
+
+  public init(flicking: Flicking): void {
+    if (this._flicking) {
+      this.destroy();
+    }
+
+    this._flicking = flicking;
+
+    flicking.on("move", this._onMove);
+    this._onMove();
```
Comment (mixed) [inline src/Fade.ts:35]: It seems that the code to initialize has changed from the constructor to the init method. Are there any issues that users switching from v3 to v4 feel? And is there no need to add a plugin guide for this?

**Review by mixed — COMMENTED** [review]
There is a change in the code at the time of initialization, but is there any issue?

File: src/Fade.ts (hunk 1 above)
Comment (WoodNeck) [reply src/Fade.ts:35]: Nope, initialization was done in the `init()` before.
https://github.com/naver/egjs-flicking-plugins/blob/master/src/Fade.ts#L16

Comment (mixed) [reply src/Fade.ts:41]: Aha. Yes. I saw the wrong one line of code.

**Review by mixed — APPROVED** [review]
LGTM

**Review by daybrush — APPROVED** [review]
LGTM

**Pull request merged**